.gitignore

```diff
@@ -20,3 +26,6 @@ scripts/docker-compose/imports/openldap/ldifs/zac-ldap-setup.ldif
 
 ### just linux things
 *.swp
+
+### GitHub ACT secrets file
+/scripts/github/.secrets
```

Comment:
For my understanding: what is this .secrets file? And why in that location?

Reply:
The location seems wrong, but this is a secrets file used by ACT. It can be setup with secrets required by the github action run with ACT.